config_example.yaml

```diff
@@ -264,3 +264,11 @@ fission:
   # minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
   # checkcert: true # check if ssl certificate of the output is valid (default: true)
   # mutualtls: false # if true, checkcert flag will be ignored (server cert will always be checked)
+
+policyreport:
+  enabled: false  # if true policyreport output is enabled
+  kubeconfig: "~/.kube/config"  # Kubeconfig file to use (only if falcosidekick is running outside the cluster)
+  failthreshold: 4 # events with priority above this threshold are mapped to fail in PolicyReport Summary and lower that those are mapped to warn (default=4)
+  maxevents: 1000 # the max number of events per report(default=1000)
+  prunebypriority: true # if true; the events with lowest severity are pruned first, in FIFO order
```

Comment:
```suggestion
  prunebypriority: false # if true; the events with lowest severity are pruned first, in FIFO order (default: false)
```

Reply:
Hey, @Issif I will make the requested changes asap and push them.
Thank you for your review!